client/app/components/CreateEventModal.jsx

```diff
@@ -0,0 +1,183 @@
+  };
+
+  constructor() {
+    super();
+
+    const initialDate = moment().format(DATE_FORMAT);
+
+    this.state = {
+      showErrorMessage: false,
+      conferenceRoomId: 1,
+      startTime:        initialDate,
+      endTime:          initialDate
+    };
+
+    _.bindAll(this,
+      ['_saveChanges', '_handleTextFieldChange', '_handleLocationChange',
+        '_handleStartTimeChange', '_handleEndTimeChange']);
+  }
+
+  render() {
+    let conferenceRoomsOptions = this.props.conferenceRooms.map(room => (
+      <option value={room.id} key={room.id}>{room.title}</option>
+    ));
+
+    return (
+      <Modal
+        show={this.props.showModal}
+        onHide={this.props.closeModal}>
+
+        <Modal.Header closeButton>
+          <Modal.Title>Create event</Modal.Title>
+        </Modal.Header>
+
+        <Modal.Body>
+          {this.state.showErrorMessage ?
+            <Alert bsStyle="danger">
+              Error occured while trying to save event.
+            </Alert>
+            : null}
+
+          <form>
+            <FormGroup>
+              <ControlLabel>Name:</ControlLabel>
+              <FormControl
+                type="text"
+                value={this.state.summary}
+                onChange={this._handleTextFieldChange}
+                name="summary" />
+            </FormGroup>
+            <FormGroup>
+              <ControlLabel>Description:</ControlLabel>
+              <FormControl
+                type="text"
+                value={this.state.description}
+                onChange={this._handleTextFieldChange}
+                name="description" />
+            </FormGroup>
+            <FormGroup>
+              <ControlLabel>Start time:</ControlLabel>
+              <DateTimeField
+                dateTime={this.state.startTime}
+                format={DATE_FORMAT}
+                inputFormat={DATE_FORMAT}
+                onChange={this._handleStartTimeChange} />
+            </FormGroup>
+            <FormGroup>
+              <ControlLabel>End time:</ControlLabel>
+              <DateTimeField
+                dateTime={this.state.endTime}
+                format={DATE_FORMAT}
+                inputFormat={DATE_FORMAT}
+                onChange={this._handleEndTimeChange} />
+            </FormGroup>
+            <FormGroup>
+              <ControlLabel>Location:</ControlLabel>
+              <FormControl
+                componentClass="select"
+                onChange={this._handleLocationChange}
+                name="location">
+                {conferenceRoomsOptions}
+              </FormControl>
+            </FormGroup>
+          </form>
+        </Modal.Body>
+
+        <Modal.Footer>
+          <Button onClick={this.props.closeModal}>Close</Button>
+          <Button
+            bsStyle="primary"
+            onClick={this._saveChanges}>
+            Save changes
+          </Button>
+        </Modal.Footer>
+      </Modal>
+    );
+  }
+
+
+  _handleTextFieldChange(e) {
```

Comment:
Did you consider moving those functions out of the class to make them truly private. You bind them to `this` in constructor anyway.